### ISSUE TITLE
Identify an indexer namespace using `namespace` + `identifier` vs just `namespace`

### ISSUE THREAD

**Issue by ra0x3** [FuelLabs/fuel-indexer#525]
- We use GraphQL `type`s to create database tables, where these tables are placed within table schema based on the indexer's `namespace` -- the tables are quite literally namespace'd based on their `namespace`.
  - https://fuellabs.github.io/fuel-indexer/master/components/assets/manifest.html
  - ⚠️ Note that SQLite does not have namespace-ing functionality like Postgres does - so this issue does not apply to SQLite
    - SQLite should only be used for demos or getting started
- The issue with this approach is it only allows users to have a single `type` per `namespace`.
  - Using our [Hello World example](https://github.com/FuelLabs/fuel-indexer/tree/master/examples/hello-world) you'll see that a user can only have one `Salutation` type in their _entire_ `fuel_examples` namespace
- We should update this functionality so that namespaces are a combination of `namespace` + `identifier`
  - This would allow users to have many similarly-named `types`s in their GraphQL schema, so long as those types are on indexers with different `identifier`s
- Basically, what we currently refer to as a `namespace` - being composed of a `namespace`, would become a `namespace` (same reference name), but would be composed of a `namespace + identifier`

Additional context
- This might apply to how we query for data as well
  - Currently we just query a namespace's data using the `/:namespace`
    - We might keep this functionality but additionally offer a query endpoint for `/:namespace/:identifier`
- This might require a new migration 

CC @deekerno for thoughts. This wouldn't be an issue for getting started or playing around. But for serious usage, I imagine you'd run into this issue.

**Comment by deekerno** [comment]
I'm in favor of this change.

In addition to allowing users to have entities of the same name in different indices, it would also bring the querying route path more in line with the index registration route path:
- the registration route -- `/api/index/:namespace/:identifier`
- the querying route -- `/api/graph/:namespace` would then be changed to `/api/graph/:namespace/:identifer`
